CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 * This file will be updated whenever a new release is put into production.
 * Any problems should be reported via the "report an issue" link in the footer of the application.
 
+## v5.1.0 - 2015-02-23
+### Fixed
+* Calendar views no longer overflow on some browsers ([#846](https://github.com/YaleSTC/reservations/issues/846)).
+* Date and time zone handling is now standardized; additionally, reservation start and due dates are now stored as `Date`s rather than as `DateTime`s ([#939](https://github.com/YaleSTC/reservations/issues/939)).
+* The handling of missed reservations was improved, and e-mails of missed open requests are no longer sent ([#1031](https://github.com/YaleSTC/reservations/issues/1031)).
+* The late checkin e-mail is no longer sent if the late fee due is zero ([#1064](https://github.com/YaleSTC/reservations/issues/1064)).
+* The extra 1px at the bottom of the navbar has been removed ([#1111](https://github.com/YaleSTC/reservations/issues/1111)).
+* Minor bugs with equipment import were fixed ([#1140](https://github.com/YaleSTC/reservations/issues/1140)).
+* Calendar cells can no longer be clicked on, preventing apparently incorrect cart updates ([#1152](https://github.com/YaleSTC/reservations/issues/1152)).
+
+### Added
+* The Bootstrap plugin was updated to v3.3, and the entire UI of the app was overhauled ([#245](https://github.com/YaleSTC/reservations/issues/245)).
```

Comment:
I wonder if we should split this one out into two things: the UI overhaul in "Added", and the Bootstrap upgrade in "Changed"? The Bootstrap upgrade is more of a "Change" than an "Addition", and that's where we have our other dependency upgrades anyway.